cmd/guacone/cmd/vulnerability.go

```diff
@@ -329,11 +332,16 @@ func searchDependencyPackages(ctx context.Context, gqlclient graphql.Client, top
 	return path, tableRows, nil
 }
 
-func queryVulnsViaVulnNodeNeighbors(ctx context.Context, gqlclient graphql.Client, topPkgResponse *model.PackagesResponse, vulnerabilityNodeID string, edgeType model.Edge, depth, pathsToReturn int) ([]string, error) {
+func queryVulnsViaVulnNodeNeighbors(ctx context.Context, gqlclient graphql.Client, topPkgResponse *model.PackagesResponse, vulnerabilityNodeID string, edgeType model.Edge, depth, pathsToReturn int) ([]string, []table.Row, error) {
 	var path []string
+	var tableRows []table.Row
 	vulnNodeNeighborResponse, err := model.Neighbors(ctx, gqlclient, vulnerabilityNodeID, []model.Edge{edgeType})
```

Comment:
so you dont need to get the neighbors from the package. You could just get it from the vulnerability. So along with `certifyVuln`, you can add `model.EdgeVulnerabilityCertifyVexStatement` here